src/ol/format/mvtformat.js

```diff
@@ -41,7 +41,7 @@ ol.format.MVT = function(opt_options) {
    * @type {ol.proj.Projection}
    */
   this.defaultDataProjection = new ol.proj.Projection({
-    code: 'EPSG:3857',
+    code: '',
```

Comment:
Not ultimately, but for now I think it makes more sense to not determine the code. Neither is it possible to change the `defaultDataProjection`, nor can pixel coordinates be reprojected. If we add support for reprojecting pixel coordinates, we can make `defaultDataProjection` configurable, and use 'EPSG:3857' as default.